WcaOnRails/db/migrate/20160627215744_create_competition_events.rb

```diff
@@ -0,0 +1,22 @@
+class CreateCompetitionEvents < ActiveRecord::Migration
+  def up
+    create_table :competition_events do |t|
+      t.string :competition_id, limit: 32, null: false
+      t.string :event_id, limit: 6, null: false
+    end
+    add_foreign_key :competition_events, :Events, column: :event_id
+    add_index :competition_events, [:competition_id, :event_id], unique: true
+
+    # Move the data to the new table.
+    Competition.all.each do |competition|
+      (competition.eventSpecs || []).split.each do |event_spec|
+        event = Event.find_by_id(event_spec.split("=")[0])
+        ActiveRecord::Base.connection.execute("insert into competition_events (competition_id, event_id) values ('#{competition.id}', '#{event.id}');")
```

Comment:
I think that in migrations @larspetrus mentioned we should use SQL in case the model changes.